dataikuapi/dss/plugin.py

```diff
@@ -177,6 +177,23 @@ def put_file(self, path, f):
         data = f.read() # eat it all, because making it work with a path variable and a MultifilePart in swing looks complicated
         return self.client._perform_empty("POST", "/plugins/%s/contents/%s" % (self.plugin_id, path), raw_body=data)
 
+    def rename_library_file(self, path, params):
+        """
+        Rename a file/folder in the plugin
+
+        :param str path: the path of the file/folder, relative ot the root of the plugin
+        :param dict params: the parameters containing the new name of the file/folder
+        """
+        return self.client._perform_empty("POST", "/plugins/%s/contents/rename/%s" % (self.plugin_id, path), body=params)
```

Comment:
Since the required structure of the params object is not obvious to someone working only with the API, we could either 
* Specify in the method description that params should have a "newName" key
* Make the method take directly the name as an argument and build the `params = {"newName": name}` in the method

Same goes for the other methods in libraries & plugins apis.
What do you think ?

Reply:
I think the second one is the better because more explicit and less prone to errors from a client perspective